### PR TITLE
Use as a microservice

### PR DESCRIPTION
**Pull request by patelnav**
I work on many AppEngine projects, one interesting way to use Ganban would be to deploy it into your AppEngine environment as a [microservice](https://cloud.google.com/appengine/docs/python/microservices-on-app-engine), alongside the project you're working on

An important things to consider in this case is to make sure you're using [namespaces](https://cloud.google.com/appengine/docs/python/multitenancy/multitenancy). This way you don't have any overlap on the application you're working on when Ganban is running in parallel.

### CODE REVIEW

**Comment by patelnav** [comment]
Another point! If you've already deployed into the default namespace, and want to transfer over to your brand new one. You can:
1) Export entities using the [Datastore Admin](https://console.cloud.google.com/datastore/settings)
2) Re-import them into the target namespace (the service name)

**Comment by patelnav** [comment]
Note, if you don't set a "service" value, or keep the value to "default", there is no change in what namespace is used

**Comment by bilby91** [comment]
@patelnav Will look at your PRs today or tomorrow morning!

File: app.yaml [hunk 1]
```diff
@@ -20,3 +21,4 @@ handlers:
 
   - url: /.*
     script: ganban.app
+    # login: admin
```
Comment (bilby91) [inline app.yaml:25]: Why is this commented ? If it's not necessary we can remove it.

**Review by bilby91 — CHANGES_REQUESTED** [review]

File: app.yaml (hunk 1 above)
Comment (patelnav) [reply app.yaml:25]: I've left it in there as a reminder that this is an option to restrict to admin, but commented it so the default behaviour is maintained. 

A completely publicly open board is not really practical. IMO it makes sense to restrict this to admin. Are there any boards that are actively being used right now?

**Review by bilby91 — APPROVED** [review]

**Comment by bilby91** [comment]
@patelnav Please resolve conflicts !

Are you interested in becoming maintainer of @ganban ?

**Comment by patelnav** [comment]
Hmm. Not sure what the conflict was. Rebased off of master and pushed again. Should be good to go!

**Comment by patelnav** [comment]
@bilby91: sure!

**Pull request merged**